.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -70,3 +70,17 @@ This change added tests and can be verified as follows:
 
   - Does this pull request introduce a new feature? (yes / no)
   - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
+
+----
+
+# Review Progress
+
+**NOTE: THE REVIEW PROGRESS MUST ONLY BE UPDATED BY AN APACHE FLINK COMMITTER!**
+
+* [ ] 1. The contribution is well-described.
+* [ ] 2. There is consensus that the contribution should go into to Flink.
```

Comment:
Can we add a point "Is blocked by feature X" or "Make sense to wait after feature Y"?

Reply:
That's a good point but I'm not sure if this belongs into the template. If the PR author is aware of the dependency, it should be noted in the PR description. If a reviewer find that dependency, it can be noted as a regular comment and the last box should not be ticked until the blocking PR was merged.